### ISSUE TITLE
Mobile menu - aligned badly and missing the menu options

### ISSUE THREAD

**Issue by corinnewo** [ji-devs/ji-cloud#3809]
The current menu on mobile looks terrible! Not aligned and missing important options.
Please align and add in the mobile menu, as in zeplin shown on issue #3539 
![mobile menu](https://user-images.githubusercontent.com/16290361/233907193-f94c64e6-7eaa-4dfa-b475-2488ce70facc.jpeg)

**Comment by MendyBerger** [comment]
Any reason this need to be a separate issue from #3539?

**Comment by corinnewo** [comment]
No, it is connected, but we need to fix, as the mobile menu is non-existent, and the current layout looks a bit shoddy! I made it separate so that this can be fixed, and not necessarily wait for the ipad fix as well.
